scripts/delete_tenant_certs.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+#
+# Copyright (c) 2019 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+TENANT_NAME=$1
+TENANT_CERTS_DIR=`pwd`/certs/$IMM_RELEASE_PREFIX/$TENANT_NAME
+
+rm -rf $TENANT_CERTS_DIR
```

Comment:
avoid - `rm -rf $VAR`, because the var can be easily set to `/`
I think it is better to put ```rm -rf `pwd`/certs/$IMM_RELEASE_PREFIX/$TENANT_NAME``` here